spec/fluent.ebnf

```diff
@@ -3,13 +3,17 @@
 Resource            ::= (blank_line | Entry | junk_line)*
 Entry               ::= Message
                       | Term
+                      | Section
                       | (ResourceComment | GroupComment | Comment)
 Message             ::= Comment? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
 Term                ::= Comment? TermIdentifier inline_space? "=" inline_space? Value Attribute* line_end
-Comment             ::= ("#" comment_line)+
+Comment             ::= (("#" | "//") comment_line)+
 GroupComment        ::= ("##" comment_line)+
 ResourceComment     ::= ("###" comment_line)+
 
+/* Sections got removed in 0.5 */
+Section             ::= "[[" inline_space? word (inline_space? word)* inline_space? "]]"
```

Comment:
I suggest adding a `section_name` production and using it here. It will simplify the `map()` operations in `grammar.mjs` for `Section`.

Reply:
Is trying for 2 hours and giving up a good-enough reason to not do this? Once this is further along, I might also take a follow-up, if that's easy for you to do. I just can't get the data out after copying and pasting working code :-(

Reply:
Sure, I can land a follow up afterwards.